services/metadata_service/api/admin.py

```diff
@@ -48,6 +52,43 @@ async def ping(self, request):
         return web.Response(text="pong", headers=MultiDict(
                                 {METADATA_SERVICE_HEADER: METADATA_SERVICE_VERSION}))
 
+    async def healthcheck(self, request):
+        """
+        ---
+        description: This end-point allow to test that service is up and
+            connected to the db
+        tags:
+        - Admin
+        produces:
+        - 'application/json'
+        responses:
+            "202":
+                description: successful operation.
+            "405":
+                description: invalid HTTP Method
+            "500":
+                description: unable to connect to DB, this node is not
+                    considered healthy and shouldn't receive traffic
+        """
+        status = {}
+        status_code = 200
+        with (
+                await AsyncPostgresDB.get_instance().pool.cursor(
+                    cursor_factory=psycopg2.extras.DictCursor
+                )
+        ) as cur:
+            await cur.execute("SELECT 1")
+            records = await cur.fetchall()
+            if len(records) > 0:
+                status["status"] = "UP"
+            else:
+                status["status"] = "DOWN"
+                status_code = 500
+
+            cur.close()
+        return web.Response(status=status_code, body=json.dumps(status))
```

Comment:
I recall having issues with writing tests for routes using the basic web.Response, as this does not set the correct content-type headers automatically. The upcoming `services.utils#web_response` helper that is used overall for json routes gets around this issue.

Seems acceptable for now, especially if nothing breaks due to the mismatched content-type header. I would suggest leaving a TODO for once the utils module and tests tooling is merged in.